app/src/main/java/org/oppia/android/app/activity/ActivityComponentImpl.kt

```diff
@@ -179,4 +180,5 @@ interface ActivityComponentImpl :
   fun inject(viewEventLogsActivity: ViewEventLogsActivity)
   fun inject(viewEventLogsTestActivity: ViewEventLogsTestActivity)
   fun inject(walkthroughActivity: WalkthroughActivity)
+  fun inject(drawableBindingAdaptersTestActivity: DrawableBindingAdaptersTestActivity)
```

Comment:
Please keep these injections in lexicographical order.

Reply:
Done.